src/test/isNotEmail.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { IsNotEmail } from '../main/isNotEmail'
+import { expect } from 'chai'
+import { validateSync } from '@hmcts/class-validator'
+
+class NotEmailTest {
+  @IsNotEmail()
+  value: any
+
+  constructor (value: any) {
+    this.value = value
+  }
+}
+
+const validContactNames = [
```

Comment:
I guess 'ContactNames' here because of the story you're playing, but it's not really relevant nomenclature for an `isNotEmail` test

Reply:
it is ContactNames because we are validation the contact name field to have no email

Reply:
updated